spec/classes/sensu_init_spec.rb

```diff
@@ -30,6 +136,106 @@
       }
     end
 
+    # FIXME: The following resource checks are only testing $sensu_etc_dir specific values
```

Comment:
These tests only checks for the $sensu_etc_dir specific details. They don't test the other file resource details that should be tested as well. Eg:

``` ruby
      it do
        should contain_file('C:/opt/sensu/bin/sensu-client.xml').with({
          'ensure'  => 'file',
          'content' => content,
          ... and other parameters....
        })
      end
```
There is still a lot of test to add for all the managed resources and functionality. But this is out of scope for this PR.